docs/index.rst

```diff
@@ -32,6 +32,28 @@ Using MediaInfo
 There isn't much to this library so instead of a lot of documentation it is
 probably best to just demonstrate how it works:
 
+Get informations on an image
+----------------------------
+
+.. code-block:: python
+
+  from pymediainfo import MediaInfo
+
+  media_info = MediaInfo.parse("/home/mdk/julien.jpg")
```

Comment:
Can we change that to a more generic location? Something like `"image.jpg"`?

Reply:
yes :)